docs/decisions/0010-taxonomy-enable-context.rst

```diff
@@ -0,0 +1,90 @@
+10. Taxonomy enabled for context
+================================
+
+Context
+-------
+
+The MVP specification says that taxonomies need to be able to be enabled/disabled for the following contexts: instance, organization, and course.
+
+Taxonomy Administrators must be able to enable a taxonomy globally for all organizations in an instance, or to set a list of organizations who can use the taxonomy.
+
+Content Authors must be able to turn taxonomies (instance and org-levels) on/off at the course level.
+
+Decision
+--------
+
+When is a taxonomy field shown to course authors in a given course?
+
++-------------+-----------------------------+--------------------------+-------------------------------+
+| tax.enabled | tax.enabled_for(course.org) | course enables all tax's | Is taxonomy shown for course? |
++=============+=============================+==========================+===============================+
+| True        | True                        | True                     | True                          |
++-------------+-----------------------------+--------------------------+-------------------------------+
+| False       | True                        | True                     | False                         |
++-------------+-----------------------------+--------------------------+-------------------------------+
+| True        | False                       | True                     | False                         |
++-------------+-----------------------------+--------------------------+-------------------------------+
+| True        | True                        | False                    | False                         |
```

Comment:
If the course hasn't enabled all taxonomies, but it has enabled this specific one, shouldn't it be shown for the course?

Reply:
I think this doesn't allow enabling taxonomies at the course level (yet)... I misunderstood this at first too, but I think `tax.enabled_for(course.org)` means `tax.enabled_for(org)` (where `org` comes from the `course`), not `tax.enabled_for(course | org)` as I first read this.